include/llarp/net.hpp

```diff
@@ -408,6 +446,12 @@ namespace llarp
       return addr4()->s_addr;
     }
 
+    sockaddr*
+    getSockAddr()
```

Comment:
I currently use an operator overload for accessing `llarp::Addr` as a `sockaddr *`

Reply:
oh ok cool, I think I had problems in the past, so I didn't bother to try.

Reply:
tried pulling it out, and got:
llarp/dns_dotlokilookup.cpp:113:42: warning: invalid user-defined conversion from ‘llarp::Addr’ to ‘sockaddr*’ [-fpermissive]